src/main/kotlin/org/rust/ide/structure/RustConstTreeElement.kt

```diff
@@ -6,7 +6,13 @@ import org.rust.lang.core.psi.RustConstItemElement
 
 class RustConstTreeElement(element: RustConstItemElement) : PsiTreeElementBase<RustConstItemElement>(element) {
 
-    override fun getPresentableText(): String? = element?.name
+    override fun getPresentableText(): String? {
+        var text = element?.name
+        if (element?.type != null) {
```

Comment:
I'm still trying to accommodate myself to idiomatic Kotlin, so thanks for the advise, it's really appreciated!